packages/website/public/schema.yml

```diff
@@ -9,6 +9,64 @@ servers:
 tags:
   - name: NFT Storage
 paths:
+  /store:
+    post:
+      tags:
+        - NFT Storage
+      summary: Store an ERC-1155 compatible NFT
+      description: |
+        Store an [ERC-1155](https://eips.ethereum.org/EIPS/eip-1155)-compatible NFT as 
+        a collection of content-addressed objects connected by IPFS CID links.
+
+        The POST request accepts `multipart/form-data` content, which must contain a form field named `meta`.
+
+        The `meta` field must contain a JSON string that conforms to the [ERC-1155 metadata schema](https://eips.ethereum.org/EIPS/eip-1155#metadata).
+
+        Any field inside the `meta` object whose value is set to `null` will be treated as a file reference, 
+        and you must supply a form data field containing the file content as a binary string.
+
+        The name of the form data field containing the file content should be the "path" of the JSON field, using `.` to traverse nested
+        objects.
+
+        For example, with a `meta` object of the form:
+
+          ```json
+          {
+            "name": "Hello",
+            "image": null,
+            "properties": {
+              "videoClip": null
+            }
+          }
+          ```
+
+        You must include form fields named `image` and `properties.videoClip` in your request body, with the content of the image and video files as the form field values.
+
+        All form fields other than `meta` must contain binary file data, and the field name must match a key in the `meta` JSON object whose value is `null`.
```

Comment:
I don't think we have `null` requirement, or if we do we probably should drop that. Maybe instead we should say if `value` been substituted is non falsy request will be rejected with error.

Reply:
> I don't think we have `null` requirement

I don't believe we do either.